modules/upgrade/pages/proxy-uyuni.adoc

```diff
@@ -51,7 +61,12 @@ spacewalk-proxy stop
 . Click menu:Software[Software Channels], and as the base channel select the openSUSE Leap {opensuse-version} channel that is listed in the [systemitem]``Customs Channels`` list.
 . In the [guimenu]``Child Channels`` pane, select the {opensuse-version} child channels.
 . Click btn:[Next], and [guimenu]``Confirm Software Channel Change`` with btn:[Confirm].
+////
```

Comment:
```suggestion
+
////
```
We'll probably need the + connector here.  If not, it could be the case that the comment would finish the list and the following item would start a new list or procedure...